src/test/cloudWatchLogs/utils.test.ts

```diff
@@ -43,9 +46,6 @@ describe('convertUriToLogGroupInfo', async function () {
 
 describe('convertLogGroupInfoToUri', function () {
     it('converts components to a valid URI', function () {
-        assert.deepStrictEqual(
-            convertLogGroupInfoToUri(goodComponents.groupName, goodComponents.streamName, goodComponents.regionName),
-            goodUri
-        )
+        assert.deepStrictEqual(createURIFromArgs(goodComponents.logGroupInfo, goodComponents.parameters), goodUri)
```

Comment:
I see, this test doesn't quite make sense in its current format because its comparing to the output to the output. I was planning on changing it to test that the URI is valid and parsable rather than equal to itself.